packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -211,7 +216,11 @@ function ContextScreens( { name, parentMenu = '', variation = '' } ) {
 			</GlobalStylesNavigationScreen>
 
 			<GlobalStylesNavigationScreen path={ parentMenu + '/colors/text' }>
-				<ScreenTextColor name={ name } variation={ variation } />
```

Comment:
@draganescu here is where I stopped using `screen-text-color.js` in favor of `screen-color-element.js` instead